test/browser-plugins/plugins.test.js

```diff
@@ -181,79 +186,185 @@ describe('Plugins', () => {
             expect(plugin._getAnonymizedProxyUrl).toBeCalled(); // eslint-disable-line
-        });
+            });
 
-        test('should use icognito context by option', async () => {
-            const plugin = new PlaywrightPlugin(playwright.chromium);
-            browserController = plugin.createController();
+            test('should use icognito context by option', async () => {
+                const plugin = new PlaywrightPlugin(playwright[browserName]);
+                const browserController = plugin.createController();
 
-            const launchContext = plugin.createLaunchContext({ useIncognitoPages: true });
+                const launchContext = plugin.createLaunchContext({ useIncognitoPages: true });
 
-            const browser = await plugin.launch(launchContext);
-            browserController.assignBrowser(browser, launchContext);
-            browserController.activate();
+                browser = await plugin.launch(launchContext);
+                browserController.assignBrowser(browser, launchContext);
+                browserController.activate();
 
-            const page = await browserController.newPage();
-            const browserContext = page.context();
-            await browserController.newPage();
+                const page = await browserController.newPage();
+                const browserContext = page.context();
+                await browserController.newPage();
 
-            expect(browserContext.pages()).toHaveLength(1);
-        });
+                expect(browserContext.pages()).toHaveLength(1);
+            });
 
-        test('should use persistent context by default', async () => {
-            const plugin = new PlaywrightPlugin(playwright.chromium);
-            browserController = plugin.createController();
+            test('should use persistent context by default', async () => {
+                const plugin = new PlaywrightPlugin(playwright[browserName]);
+                const browserController = plugin.createController();
 
-            const launchContext = plugin.createLaunchContext();
+                const launchContext = plugin.createLaunchContext();
 
-            const browser = await plugin.launch(launchContext);
-            browserController.assignBrowser(browser, launchContext);
-            browserController.activate();
+                browser = await plugin.launch(launchContext);
+                browserController.assignBrowser(browser, launchContext);
+                browserController.activate();
 
-            const page = await browserController.newPage();
-            const context = await page.context();
-            await browserController.newPage();
+                const page = await browserController.newPage();
+                const context = await page.context();
+                await browserController.newPage();
 
-            expect(context.pages()).toHaveLength(3);
-        });
+                expect(context.pages()).toHaveLength(3);
```

Comment:
Would deserve a comment why 3 pages. Is it because of the typical `about:blank`?